src/FitterBase.cc

```diff
@@ -237,9 +242,13 @@ MinimizeResult FitterBase::minimize(std::string const &whatToFit, double nSigmaC
         MeasuredStarList msOutliers;
         FittedStarList fsOutliers;
         // keep nOutliers so we don't have to sum msOutliers.size()+fsOutliers.size() twice below.
-        std::size_t nOutliers = findOutliers(nSigmaCut, msOutliers, fsOutliers);
+        std::size_t nOutliers = findOutliers(nSigmaCut, msOutliers, fsOutliers, sigmaCut);
+        if ((sigmaRelTolerance > 0) && (oldSigmaCut > 0 && (1 - sigmaCut / oldSigmaCut) < sigmaRelTolerance)){
```

Comment:
I think this needs a comment about the nature of this percent calculation, and a log message if we `break` (probably INFO level, since it only would happen once per `minimize()` call) noting the actual percent change and chi2 value that caused the loop to exit.